crane/hooks/__init__.py

```diff
@@ -4,10 +4,18 @@
 
 HOOKS = []
 
+is_active = True
 
 def dispatch(event):
+    if not is_active:
+        return
+
     if not HOOKS:
         HOOKS.extend(getattr(module, 'Hook')() for module in AVAILABLE_HOOKS)
 
     for hook in HOOKS:
         hook.dispatch(event)
+
+def deactivate():
+    global is_active
```

Comment:
`global` makes this create a new variable shared across all modules, you should use `nonlocal` instead which instructs python to modify the variable from the outer scope